rcsb/utils/io/StashUtil.py

```diff
@@ -83,6 +83,10 @@ def storeBundle(self, url, remoteDirPath, remoteStashPrefix="A", userName=None,
                 if ok:
                     remotePath = os.path.join("/", remoteDirPath, fn)
                     ok = sftpU.put(self.__localStashTarFilePath, remotePath)
+            elif url.startswith("/") or url.startswith("file"):
+                fileU = FileUtil()
+                remotePath = os.path.join(url, remoteDirPath, fn)
+                ok = fileU.put(self.__localStashTarFilePath, remotePath)
             elif not url:
```

Comment:
I would move the `elif not url:` to the top of this if block, since that would catch the absence right away and then you wouldn't need to add `if url` to the beginning of each if statement (otherwise, if you leave things in this order, you will need to change your line to `elif url and (url.startswith("/") or url.startswith("file")):`).